src/sub/payments.py

```diff
@@ -151,11 +153,18 @@ def cancel_subscription(uid: str, sub_id: str) -> FlaskResponse:
                 sub_id, utils.get_indempotency_key()
             )
             updated_customer = fetch_customer(g.subhub_account, uid)
-            logger.info("updated customer", updated_customer=updated_customer)
+            logger.debug("updated customer", updated_customer=updated_customer)
             subs = retrieve_stripe_subscriptions(updated_customer)
-            logger.info("subs", subs=subs, type=type(subs))
+            logger.debug("retrieved subs", subs=subs, type=type(subs))
             for sub in subs:
                 if sub["cancel_at_period_end"] and sub["id"] == sub_id:
+                    logger.debug(
+                        "cancel subscription response",
+                        response_message={
```

Comment:
store `response_message` as a variable with the dict values, then use it here and the return statement below.